src/Paramore.Brighter.MessagingGateway.AzureServiceBus/AzureServiceBusWrappers/ServiceBusSenderWrapper.cs

```diff
@@ -69,9 +69,9 @@ public void Close()
             _serviceBusSender.CloseAsync().Wait();
         }
 
-        public async Task CloseAsync()
+        public Task CloseAsync()
         {
-            await _serviceBusSender.CloseAsync();
+            return _serviceBusSender.CloseAsync();
```

Comment:
i elide  the task. hence removing the async state machine

Reply:
Follow up question as to why we are eliding the task?

Reply:
no async state machine is created so

 * better on memory usage since 
 * less il in the assembly so smaller dll and faster git
 * faster

all of these a very small incremental improvements. but given the code is essentially equivalent i like to leverage them